repose-aggregator/components/filters/keystone-v2-filter/src/main/resources/META-INF/schema/config/keystone-v2-common.xsd

```diff
@@ -129,6 +132,17 @@
             </xs:element>
         </xs:sequence>
 
+        <xs:attribute name="header-extraction-name" type="xs:string" use="optional">
```

Comment:
I wasn't sure if we wanted to default this attribute or not. Right now, I left it with no default since it's purely optional -- the user can choose to extract the tenant ID either from the URI, a header, or both. If/when we do drop support for URI extraction in this filter, we will probably want a default.

Reply:
I don't think i like this as an attribute still. I series of elements much like the uri regex make more sense to me. The other way is they do a header translation and/or merge header filter to put a bunch of values together, which while very much the toolbox approach feels a little klugey.